packages/apps-config/src/endpoints/productionRelayPolkadot.ts

```diff
@@ -499,6 +499,16 @@ export const prodParasPolkadot: EndpointOption[] = [
       logo: nodesParallelSVG
     }
   },
+  {
+    homepage: 'https://peaq.network/',
+    info: 'peaq',
+    paraId: 3338,
+    text: 'Peaq',
+    providers: {},
```

Comment:
```suggestion
    providers: {},
    text: 'Peaq',
```

This should fix the linting issues.